### ISSUE TITLE
Console wallet not allowing coinbase UTXOs to be spent

### ISSUE THREAD

**Issue by hansieodendaal** [tari-project/tari#2443]
**Describe the bug**

Only one transfer of all initial coinbase transactions that were successfully detected a mined on the base layer was allowed. After a restart and after more successful mining, consecutive transfer transactions were not allowed, even with multiple restarts.
(_Tested with build `435c335`_)

**To Reproduce**

- Perform merge mining and collect mined coinbase transactions in the wallet.
- Stop merge mining.
- Spend half of these coinbase UTXOs.
- Shut down the console wallet.
- Restart the console wallet and try to spend the remaining coinbase UTXOs.

**Expected behaviour**

All mined coinbase transactions should be spendable.

**Screenshots**

![image](https://user-images.githubusercontent.com/39146854/99668735-cb490580-2a76-11eb-9738-d223f7080972.png)

See more screen shots below.

**Desktop (please complete the following information):**

Windows, Linux

**Smartphone (please complete the following information):**

n/a

**Additional context**

These files show the exact use case:
- Screen shots: [console_wallet_transactions.pdf](https://github.com/tari-project/tari/files/5572259/console_wallet_transactions.pdf)
- Console wallet trace log file: [base_layer.zip](https://github.com/tari-project/tari/files/5572260/base_layer.zip)
- Mining proxy debug log file: [proxy.zip](https://github.com/tari-project/tari/files/5572303/proxy.zip)
- Console wallet SQL database: [console-wallet.zip](https://github.com/tari-project/tari/files/5572290/console-wallet.zip)

**Comment by delta1** [comment]
Very interesting - do you have the wallet logs please? Did this only happen with coinbase UTXOs or all?

**Comment by hansieodendaal** [comment]
> Very interesting - do you have the wallet logs please? Did this only happen with coinbase UTXOs or all?

Unfortunately not, but it is easy to make new logs

**Comment by delta1** [comment]
Will try it out

**Comment by hansieodendaal** [comment]
Also a problem with calculating balances. When copying the console wallet to a base node wallet:
```
>> get-balance
>> Balances:
Available balance: 447343.079685 T
Pending incoming balance: 0 µT
Pending outgoing balance: 0 µT

>> list-utxos
>>
You have 82 UTXOs: (value, commitment, mature in ? blocks, flags)
   2259 µT, e84c166905b98643a529a5d1a21f06746aa84ffd8424cd88d99aa5b9c9ec5e3d,   0, (empty)
   5522.525203 T, 88be2884a28cfd6e1dc9975e99993138e08e5406ea690fa08bab2b1ff765fe73,   0, COINBASE_OUTPUT
   5522.527629 T, 605aab6a108d51583679fa78e8808241c23e751b52f5feac3d5e79861d9df928,   0, COINBASE_OUTPUT
   5522.530055 T, 8cfcf69b58576be95bd0f42f3d47a0c56b2620709ac786f88504a8f693fea573,   0, COINBASE_OUTPUT
   5522.559167 T, e80b48366edceb98e38d2fff311200005227d968e9d7633cdc51345c0178f35b,   0, COINBASE_OUTPUT
   5522.561593 T, f80c53011845e070d82967e7974acbd27aa9821f0655ec27c4aa7ccfa39ec558,   0, COINBASE_OUTPUT
   5522.564019 T, 9a670673a6bfb7c42b47506c903f0aeac20ec9d29cf206d8856ba1ad36acd21d,   0, COINBASE_OUTPUT
   5522.566445 T, 6a9c0fbc768d30eb30d3ebf51c05d9c21794ca88d99cc64609263d9b3b6b2c30,   0, COINBASE_OUTPUT
   5522.573723 T, a2622ad359be6bec2b4dfb2794659d8cea9cec34eaf874826a09553a6d0cc732,   0, COINBASE_OUTPUT
   5522.576149 T, f8531ca3fcd16e62e62bcecd9ad6629b1301483d5ae52e338aea3ac9c60e0328,   0, COINBASE_OUTPUT
   5522.583427 T, 06a915ffecf275fbd5cde1988e1f751c42fb9047d109fa8b3264cb4981e19b2c,   0, COINBASE_OUTPUT
   5522.593131 T, ec6961c760968f162dad5d3bb07a1c469b432a55c491b409081022226c6f483d,   0, COINBASE_OUTPUT
   5522.597983 T, 8a3084130750169a551dee68a7f4ca9dd395a0870c84eec42a877ec08df2cf2f,   0, COINBASE_OUTPUT
   5522.598779 T, 1eb00639e0d32390c08e69bcaa655608d37313c28f442f352941b5c91b4e4020,   0, COINBASE_OUTPUT
   5522.605261 T, 4278cbcb349af2b70831a664150f72a8f9f3d7f8eef4a67c30c77e1ff46c1b78,   0, COINBASE_OUTPUT
   5522.607687 T, a67904ce2c082722d3c632fef74d2a51ede042d202102c8dafa3814a1057e10a,   0, COINBASE_OUTPUT
   5522.610113 T, 1449cfb9217f99dfd7ec0723c3c6b2c494044e07b35f4614f37c9237e0a73760,   0, COINBASE_OUTPUT
   5522.612539 T, 84de491da88557d7be4c72c2b5f6c739949374747280c2bb0cc1701edfddae58,   0, COINBASE_OUTPUT
   5522.614965 T, 88c20c68cf0b54cd353ce5fe3d78c3c4b19f76ec6b9f8de5516e28bcb9bb5217,   0, COINBASE_OUTPUT
   5522.617391 T, ae7cd4bc9afe0a423c5d38c30fc58dec3c7ffe48bbbca51943dab2b0feb31d7b,   0, COINBASE_OUTPUT
   5522.619817 T, 84359a20efdc7e5466a25836470e032ae96db0360a6c879d620b301d2f42c733,   0, COINBASE_OUTPUT
   5522.622243 T, 326c7aacb98f9bf82856cb2407df5fb781cd92c1cf936fd375e8d30dd8ba0478,   0, COINBASE_OUTPUT
   5522.624669 T, 8e3861c1226cbba1849276c76a39fbccd89e62599305881f7a06ccce49e5487a,   0, COINBASE_OUTPUT
   5522.627095 T, 54528df0f4df98ef7c2cf51cfb546be6382ff44971c6c2cf89d773f31019c146,   0, COINBASE_OUTPUT
   5522.629521 T, bc68e98376ae3cbbda161d7b2646259bc3f9f99e8ea1e351c49104aec4213509,   0, COINBASE_OUTPUT
   5522.634373 T, 20bba5e89a7c987cf9987433a7dd2604863001417528c9f8841c14f34f63452c,   0, COINBASE_OUTPUT
   5522.639225 T, f2eb2665b85f0d87a0314eccb0c149089bbc635a45733965bba5b7761b4d1f3a,   0, COINBASE_OUTPUT
   5522.641651 T, 3243bd1c9ab3ad604c346b8e4f13b850fcd830ba4f17e5f050fa900db9b28e39,   0, COINBASE_OUTPUT
   5522.644077 T, aca659bb7a7b4c53606e8cf508d0857ddaddb4134279d570308cb1159d847660,   0, COINBASE_OUTPUT
   5522.646503 T, 30851f693de8c3112b811d9a42aa90c6c6bfff5ff43fe99427a8779c8091296e,   0, COINBASE_OUTPUT
   5522.648929 T, b484637bde51fe8587b05bb71be81600eb2f80a585b896c85aed4973d58d6f5b,   0, COINBASE_OUTPUT
   5522.651355 T, 44a4be80c4355ea2fdc9913a7b4462da968c9f8ecf81a108f04240e9a07fa253,   0, COINBASE_OUTPUT
   5522.653781 T, 0248403ebde28b5b357338be4853695bd3cc3515e0d9ba33127bdd9e032c9650,   0, COINBASE_OUTPUT
   5522.656207 T, 4ee2c98ae73d9c2e0bda84c01685543e63c1a27584ab3d09fd42943eb2e3a31b,   0, COINBASE_OUTPUT
   5522.658633 T, 74b31cb24b20e4c8a8463306c24dedebaca5ffb498c15318dda269019e565111,   0, COINBASE_OUTPUT
   5522.661059 T, 8a25b39982255060db60a3377590f20cf7aa69b2271f429e6e1d0709ef818962,   0, COINBASE_OUTPUT
   5522.673189 T, ae27afc36e06baccca756c36581e5a7ffbc7a941b00983eb817929b55e55e859,   0, COINBASE_OUTPUT
   5522.678041 T, 5c86a99d0691637163f6b506d64417bea331f26856332d591effeb11151c0132,   0, COINBASE_OUTPUT
   5522.680467 T, 02d1132d932e2f3db0c81e07b02cc20b5c2384b43cfe6ef96e784b1c7eddde12,   0, COINBASE_OUTPUT
   5522.693288 T, 3ead2aa301b69e0b8c7d02ad824bbe8f20aa2700efd0a32ef97b9ffc4f5f6e58,   0, COINBASE_OUTPUT
   5522.695023 T, 4c7b27167bff9c29a0c3aaf0a7a96bf46172b24be255ec95f92a440f9dce107d,   0, COINBASE_OUTPUT
   5522.699875 T, f0211240ba3a65ff7a9ce81c6f05e4ac80ee576f50ac50768dbfa746f7e5fa0c,   0, COINBASE_OUTPUT
   5522.702301 T, a2a842c3084846d6a3ac930c61e665d1417af8216dc21f38978a04771fdd206f,   0, COINBASE_OUTPUT
   5522.704727 T, f0d3fe3924f9acbe04772a196be3228aa4ee1efb49cd5d7a193a5d866b3ee85f,   0, COINBASE_OUTPUT
   5522.709579 T, 9c3ea8ea43a9873db728c2202feaf4d7b65d8dc58af299552002cfd44ec26f74,   0, COINBASE_OUTPUT
   5522.714431 T, 02dfd50059df9b0724c429f5f3545bf3dbe70a38547174b843959809315b6c0e,   0, COINBASE_OUTPUT
   5522.716857 T, 42308b355b8c25f5f6ed311d2d4b0ed0f1d44fa67f3fdd4a55fffd38d1c44040,   0, COINBASE_OUTPUT
   5522.719283 T, 262f7c3f06380ab361f4fe0cb67984a408742a21d0c5725061a740b94cb9d678,   0, COINBASE_OUTPUT
   5522.724135 T, c8f883c0c8befdbb81a3cfd9f8a66fb326e498fd882cc27b9ef6140bfd9a560a,   0, COINBASE_OUTPUT
   5522.729284 T, 38e8682b6e7ee3720abebbe055bbf2fb00f13d8deef3b525160d8c37ab2fa00e,   0, COINBASE_OUTPUT
   5522.731413 T, 0cf06ec7782a020bedf48859b3f942322fc8e15977e06cf8bb92036803a61e6c,   0, COINBASE_OUTPUT
   5522.733839 T, e06db711d25a468ffcab458e04af7bc28ccd03c40422b893ca3b98216e2ddb6d,   0, COINBASE_OUTPUT
   5522.736265 T, 0426b020bfd5315b637b41c55afd79d08fe95ea1f8b4b8f32a4fb98d2f288458,   0, COINBASE_OUTPUT
   5522.738691 T, 0c819219bd72c0051a7a7d6af6ebb17fcba9de4ef226cbd66758fe60a8f7eb2b,   0, COINBASE_OUTPUT
   5522.741117 T, baef7ff8d86805b057b578fa5a389c3f0a04d7b6599777a6ffeabe36d0000109,   0, COINBASE_OUTPUT
   5522.743543 T, e06cbb5c1adfaba5d2481d7d69b74d1c48463c9bb2ed6febcbe67ded5a530047,   0, COINBASE_OUTPUT
   5522.745969 T, 8c3c61b4a7a575354dabf882adb9e307411a40babd4277f6920933be2a5d8c5a,   0, COINBASE_OUTPUT
   5522.748395 T, 70f0e681f9e9dbe2ad45f7d80e075872ddaa85f18749a3a9bf6991509ede4f37,   0, COINBASE_OUTPUT
   5522.750821 T, 6673656a58abdf95bb5d38243c63181ba3f18c72620a3dc6940c9d417b681c3e,   0, COINBASE_OUTPUT
   5522.753247 T, f82dca1473f62d587a9c9ea3858a9caa64b6404655f389d15832bf65f4cb2c27,   0, COINBASE_OUTPUT
   5522.755673 T, f6a7a762f0e21a47ee507e6ee8a073455eecb01948e06f103a4d3c61d4821620,   0, COINBASE_OUTPUT
   5522.758099 T, d635dcb84de16e145d981730e9a4dce91fdc0acc2de976d70bb8ab5273e51f7f,   0, COINBASE_OUTPUT
   5522.760525 T, a675800caf2792aca44222686eeab289d894d474b5f092d0597eca26965a6854,   0, COINBASE_OUTPUT
   5522.765377 T, 9ede4dc24537daf08800522812c1a7b99b619ebd24a5e9737554eb6772f46259,   0, COINBASE_OUTPUT
   5522.767803 T, c4615b416df7ea8aab57cc54a0e6301d2a339f5db14fc7c88cee7bde7f7c6608,   0, COINBASE_OUTPUT
   5522.770229 T, 78048308b8f74f4da0915c82e3906a824b40a360ad45404518e88a555c00ee30,   0, COINBASE_OUTPUT
   5522.772655 T, 103d27ebe8bbf4ed56d3f3afdbea1c5be040616b50b4613a6634974b5194970c,   0, COINBASE_OUTPUT
   5522.775081 T, c480b05689cc0d3407f9bbe7b10957da7e85f9d4b295696b80a9708f0c49df13,   0, COINBASE_OUTPUT
   5522.777507 T, 446c717834eca91ab09aed5a5ebd6741ef2fbb220e1445cce3a7b641d5570439,   0, COINBASE_OUTPUT
   5522.779933 T, 9c1f7255725a9c1709f4f0e0008e7486bb8d08c106a8983f8c5a236ab5581a51,   0, COINBASE_OUTPUT
   5522.790397 T, 6a8610bc79b2348f70fe4fbfd8181c942633f98c4507d3c9368662dfcec13775,   0, COINBASE_OUTPUT
   5523.242098 T, c89ed9491159387488e763c4a27df46b646bc4b08a146421d4b4db4899b8d77e,   0, COINBASE_OUTPUT
   5523.245725 T, c402c02ff5510e2f4b803c8d9e9a04ece184629e5b86ef26cb16a8c5c298fd77,   0, COINBASE_OUTPUT
   5523.257855 T, 6213ca17a149027ce41cc08b9a19fff1f929f42bc8710db815a594c13f336206,   0, COINBASE_OUTPUT
   5523.272411 T, bc3704e8589d7a296bcd91fbe47701b81993b0a792c48621968d3efd2f4bbe21,   0, COINBASE_OUTPUT
   5523.274837 T, 944f25d4cbddeb73b5bb397e82ee3142a66b12124e1c418c8d932881f8281333,   0, COINBASE_OUTPUT
   5523.282115 T, 2e997831d9e7afb743219d54275b75ef7a89da0435feb62746d592efa39a2b16,   0, COINBASE_OUTPUT
   5523.284541 T, a07b4f404242ce0b1e1ccdc492b82d1368943ecfca38a65c57a8013b4c81f23f,   0, COINBASE_OUTPUT
   5523.289393 T, 1a18321ef7d92f4467820514b7bfd13143fabf784c80b901586452f9990b7e14,   0, COINBASE_OUTPUT
   5523.296671 T, ba4bf550d8dda2987ed175295128133056f6a54e76cec1af8ff567ebabb68d5e,   0, COINBASE_OUTPUT
   5523.303949 T, 027c9dd268ef967ff688c1ebd48cdabdbec2b76f62ecd6a2625726db6cb98259,   0, COINBASE_OUTPUT
   5523.306375 T, 087294b16055be95eae25f1572d15e28c561d8d44c2bcc375934ea2b63ced42f,   0, COINBASE_OUTPUT


>> get-balance
>> Balances:
Available balance: 447343.079685 T
Pending incoming balance: 0 µT
Pending outgoing balance: 0 µT

>> get-chain-metadata
Height of longest chain : 6798
Geometric mean of longest chain : 120570835773975006132000
Best block : 91b8ffbb3a2bf2905dff1b215c719d3ca14e79c4221c2b4c77c23185f2275ffd
Pruning horizon : 0
Effective pruned height : 0

>>
>> send-tari 100000T 5a779145a2a26edb0f70abed8ff4c82a549115b4af2440a6bc89721be6186f30
>> Sending 100000.000000 T Tari to 5a779145a2a26edb0f70abed8ff4c82a549115b4af2440a6bc89721be6186f30

>> send-tari 100000T 5a779145a2a26edb0f70abed8ff4c82a549115b4af2440a6bc89721be6186f30
>> Sending 100000.000000 T Tari to 5a779145a2a26edb0f70abed8ff4c82a549115b4af2440a6bc89721be6186f30

>> send-tari 100000T 5a779145a2a26edb0f70abed8ff4c82a549115b4af2440a6bc89721be6186f30
>> Sending 100000.000000 T Tari to 5a779145a2a26edb0f70abed8ff4c82a549115b4af2440a6bc89721be6186f30

>> send-tari 100000T 5a779145a2a26edb0f70abed8ff4c82a549115b4af2440a6bc89721be6186f30
>> Sending 100000.000000 T Tari to 5a779145a2a26edb0f70abed8ff4c82a549115b4af2440a6bc89721be6186f30

>> get-balance
>> Balances:
Available balance: 430774.748830 T
Pending incoming balance: 19752.845334 T
Pending outgoing balance: 121506.529249 T

>> list-utxos
>>
You have 79 UTXOs: (value, commitment, mature in ? blocks, flags)
   2259 µT, e84c166905b98643a529a5d1a21f06746aa84ffd8424cd88d99aa5b9c9ec5e3d,   0, (empty)
   5522.525203 T, 88be2884a28cfd6e1dc9975e99993138e08e5406ea690fa08bab2b1ff765fe73,   0, COINBASE_OUTPUT
   5522.527629 T, 605aab6a108d51583679fa78e8808241c23e751b52f5feac3d5e79861d9df928,   0, COINBASE_OUTPUT
   5522.530055 T, 8cfcf69b58576be95bd0f42f3d47a0c56b2620709ac786f88504a8f693fea573,   0, COINBASE_OUTPUT
   5522.559167 T, e80b48366edceb98e38d2fff311200005227d968e9d7633cdc51345c0178f35b,   0, COINBASE_OUTPUT
   5522.561593 T, f80c53011845e070d82967e7974acbd27aa9821f0655ec27c4aa7ccfa39ec558,   0, COINBASE_OUTPUT
   5522.564019 T, 9a670673a6bfb7c42b47506c903f0aeac20ec9d29cf206d8856ba1ad36acd21d,   0, COINBASE_OUTPUT
   5522.566445 T, 6a9c0fbc768d30eb30d3ebf51c05d9c21794ca88d99cc64609263d9b3b6b2c30,   0, COINBASE_OUTPUT
   5522.573723 T, a2622ad359be6bec2b4dfb2794659d8cea9cec34eaf874826a09553a6d0cc732,   0, COINBASE_OUTPUT
   5522.576149 T, f8531ca3fcd16e62e62bcecd9ad6629b1301483d5ae52e338aea3ac9c60e0328,   0, COINBASE_OUTPUT
   5522.583427 T, 06a915ffecf275fbd5cde1988e1f751c42fb9047d109fa8b3264cb4981e19b2c,   0, COINBASE_OUTPUT
   5522.593131 T, ec6961c760968f162dad5d3bb07a1c469b432a55c491b409081022226c6f483d,   0, COINBASE_OUTPUT
   5522.597983 T, 8a3084130750169a551dee68a7f4ca9dd395a0870c84eec42a877ec08df2cf2f,   0, COINBASE_OUTPUT
   5522.598779 T, 1eb00639e0d32390c08e69bcaa655608d37313c28f442f352941b5c91b4e4020,   0, COINBASE_OUTPUT
   5522.605261 T, 4278cbcb349af2b70831a664150f72a8f9f3d7f8eef4a67c30c77e1ff46c1b78,   0, COINBASE_OUTPUT
   5522.607687 T, a67904ce2c082722d3c632fef74d2a51ede042d202102c8dafa3814a1057e10a,   0, COINBASE_OUTPUT
   5522.610113 T, 1449cfb9217f99dfd7ec0723c3c6b2c494044e07b35f4614f37c9237e0a73760,   0, COINBASE_OUTPUT
   5522.612539 T, 84de491da88557d7be4c72c2b5f6c739949374747280c2bb0cc1701edfddae58,   0, COINBASE_OUTPUT
   5522.614965 T, 88c20c68cf0b54cd353ce5fe3d78c3c4b19f76ec6b9f8de5516e28bcb9bb5217,   0, COINBASE_OUTPUT
   5522.617391 T, ae7cd4bc9afe0a423c5d38c30fc58dec3c7ffe48bbbca51943dab2b0feb31d7b,   0, COINBASE_OUTPUT
   5522.619817 T, 84359a20efdc7e5466a25836470e032ae96db0360a6c879d620b301d2f42c733,   0, COINBASE_OUTPUT
   5522.622243 T, 326c7aacb98f9bf82856cb2407df5fb781cd92c1cf936fd375e8d30dd8ba0478,   0, COINBASE_OUTPUT
   5522.624669 T, 8e3861c1226cbba1849276c76a39fbccd89e62599305881f7a06ccce49e5487a,   0, COINBASE_OUTPUT
   5522.627095 T, 54528df0f4df98ef7c2cf51cfb546be6382ff44971c6c2cf89d773f31019c146,   0, COINBASE_OUTPUT
   5522.629521 T, bc68e98376ae3cbbda161d7b2646259bc3f9f99e8ea1e351c49104aec4213509,   0, COINBASE_OUTPUT
   5522.634373 T, 20bba5e89a7c987cf9987433a7dd2604863001417528c9f8841c14f34f63452c,   0, COINBASE_OUTPUT
   5522.639225 T, f2eb2665b85f0d87a0314eccb0c149089bbc635a45733965bba5b7761b4d1f3a,   0, COINBASE_OUTPUT
   5522.641651 T, 3243bd1c9ab3ad604c346b8e4f13b850fcd830ba4f17e5f050fa900db9b28e39,   0, COINBASE_OUTPUT
   5522.644077 T, aca659bb7a7b4c53606e8cf508d0857ddaddb4134279d570308cb1159d847660,   0, COINBASE_OUTPUT
   5522.646503 T, 30851f693de8c3112b811d9a42aa90c6c6bfff5ff43fe99427a8779c8091296e,   0, COINBASE_OUTPUT
   5522.648929 T, b484637bde51fe8587b05bb71be81600eb2f80a585b896c85aed4973d58d6f5b,   0, COINBASE_OUTPUT
   5522.651355 T, 44a4be80c4355ea2fdc9913a7b4462da968c9f8ecf81a108f04240e9a07fa253,   0, COINBASE_OUTPUT
   5522.653781 T, 0248403ebde28b5b357338be4853695bd3cc3515e0d9ba33127bdd9e032c9650,   0, COINBASE_OUTPUT
   5522.656207 T, 4ee2c98ae73d9c2e0bda84c01685543e63c1a27584ab3d09fd42943eb2e3a31b,   0, COINBASE_OUTPUT
   5522.658633 T, 74b31cb24b20e4c8a8463306c24dedebaca5ffb498c15318dda269019e565111,   0, COINBASE_OUTPUT
   5522.661059 T, 8a25b39982255060db60a3377590f20cf7aa69b2271f429e6e1d0709ef818962,   0, COINBASE_OUTPUT
   5522.673189 T, ae27afc36e06baccca756c36581e5a7ffbc7a941b00983eb817929b55e55e859,   0, COINBASE_OUTPUT
   5522.678041 T, 5c86a99d0691637163f6b506d64417bea331f26856332d591effeb11151c0132,   0, COINBASE_OUTPUT
   5522.680467 T, 02d1132d932e2f3db0c81e07b02cc20b5c2384b43cfe6ef96e784b1c7eddde12,   0, COINBASE_OUTPUT
   5522.693288 T, 3ead2aa301b69e0b8c7d02ad824bbe8f20aa2700efd0a32ef97b9ffc4f5f6e58,   0, COINBASE_OUTPUT
   5522.695023 T, 4c7b27167bff9c29a0c3aaf0a7a96bf46172b24be255ec95f92a440f9dce107d,   0, COINBASE_OUTPUT
   5522.699875 T, f0211240ba3a65ff7a9ce81c6f05e4ac80ee576f50ac50768dbfa746f7e5fa0c,   0, COINBASE_OUTPUT
   5522.702301 T, a2a842c3084846d6a3ac930c61e665d1417af8216dc21f38978a04771fdd206f,   0, COINBASE_OUTPUT
   5522.704727 T, f0d3fe3924f9acbe04772a196be3228aa4ee1efb49cd5d7a193a5d866b3ee85f,   0, COINBASE_OUTPUT
   5522.709579 T, 9c3ea8ea43a9873db728c2202feaf4d7b65d8dc58af299552002cfd44ec26f74,   0, COINBASE_OUTPUT
   5522.714431 T, 02dfd50059df9b0724c429f5f3545bf3dbe70a38547174b843959809315b6c0e,   0, COINBASE_OUTPUT
   5522.716857 T, 42308b355b8c25f5f6ed311d2d4b0ed0f1d44fa67f3fdd4a55fffd38d1c44040,   0, COINBASE_OUTPUT
   5522.719283 T, 262f7c3f06380ab361f4fe0cb67984a408742a21d0c5725061a740b94cb9d678,   0, COINBASE_OUTPUT
   5522.724135 T, c8f883c0c8befdbb81a3cfd9f8a66fb326e498fd882cc27b9ef6140bfd9a560a,   0, COINBASE_OUTPUT
   5522.729284 T, 38e8682b6e7ee3720abebbe055bbf2fb00f13d8deef3b525160d8c37ab2fa00e,   0, COINBASE_OUTPUT
   5522.731413 T, 0cf06ec7782a020bedf48859b3f942322fc8e15977e06cf8bb92036803a61e6c,   0, COINBASE_OUTPUT
   5522.733839 T, e06db711d25a468ffcab458e04af7bc28ccd03c40422b893ca3b98216e2ddb6d,   0, COINBASE_OUTPUT
   5522.736265 T, 0426b020bfd5315b637b41c55afd79d08fe95ea1f8b4b8f32a4fb98d2f288458,   0, COINBASE_OUTPUT
   5522.738691 T, 0c819219bd72c0051a7a7d6af6ebb17fcba9de4ef226cbd66758fe60a8f7eb2b,   0, COINBASE_OUTPUT
   5522.741117 T, baef7ff8d86805b057b578fa5a389c3f0a04d7b6599777a6ffeabe36d0000109,   0, COINBASE_OUTPUT
   5522.743543 T, e06cbb5c1adfaba5d2481d7d69b74d1c48463c9bb2ed6febcbe67ded5a530047,   0, COINBASE_OUTPUT
   5522.745969 T, 8c3c61b4a7a575354dabf882adb9e307411a40babd4277f6920933be2a5d8c5a,   0, COINBASE_OUTPUT
   5522.748395 T, 70f0e681f9e9dbe2ad45f7d80e075872ddaa85f18749a3a9bf6991509ede4f37,   0, COINBASE_OUTPUT
   5522.750821 T, 6673656a58abdf95bb5d38243c63181ba3f18c72620a3dc6940c9d417b681c3e,   0, COINBASE_OUTPUT
   5522.753247 T, f82dca1473f62d587a9c9ea3858a9caa64b6404655f389d15832bf65f4cb2c27,   0, COINBASE_OUTPUT
   5522.755673 T, f6a7a762f0e21a47ee507e6ee8a073455eecb01948e06f103a4d3c61d4821620,   0, COINBASE_OUTPUT
   5522.758099 T, d635dcb84de16e145d981730e9a4dce91fdc0acc2de976d70bb8ab5273e51f7f,   0, COINBASE_OUTPUT
   5522.760525 T, a675800caf2792aca44222686eeab289d894d474b5f092d0597eca26965a6854,   0, COINBASE_OUTPUT
   5522.765377 T, 9ede4dc24537daf08800522812c1a7b99b619ebd24a5e9737554eb6772f46259,   0, COINBASE_OUTPUT
   5522.770229 T, 78048308b8f74f4da0915c82e3906a824b40a360ad45404518e88a555c00ee30,   0, COINBASE_OUTPUT
   5522.775081 T, c480b05689cc0d3407f9bbe7b10957da7e85f9d4b295696b80a9708f0c49df13,   0, COINBASE_OUTPUT
   5522.777507 T, 446c717834eca91ab09aed5a5ebd6741ef2fbb220e1445cce3a7b641d5570439,   0, COINBASE_OUTPUT
   5522.779933 T, 9c1f7255725a9c1709f4f0e0008e7486bb8d08c106a8983f8c5a236ab5581a51,   0, COINBASE_OUTPUT
   5523.242098 T, c89ed9491159387488e763c4a27df46b646bc4b08a146421d4b4db4899b8d77e,   0, COINBASE_OUTPUT
   5523.245725 T, c402c02ff5510e2f4b803c8d9e9a04ece184629e5b86ef26cb16a8c5c298fd77,   0, COINBASE_OUTPUT
   5523.257855 T, 6213ca17a149027ce41cc08b9a19fff1f929f42bc8710db815a594c13f336206,   0, COINBASE_OUTPUT
   5523.272411 T, bc3704e8589d7a296bcd91fbe47701b81993b0a792c48621968d3efd2f4bbe21,   0, COINBASE_OUTPUT
   5523.274837 T, 944f25d4cbddeb73b5bb397e82ee3142a66b12124e1c418c8d932881f8281333,   0, COINBASE_OUTPUT
   5523.282115 T, 2e997831d9e7afb743219d54275b75ef7a89da0435feb62746d592efa39a2b16,   0, COINBASE_OUTPUT
   5523.284541 T, a07b4f404242ce0b1e1ccdc492b82d1368943ecfca38a65c57a8013b4c81f23f,   0, COINBASE_OUTPUT
   5523.289393 T, 1a18321ef7d92f4467820514b7bfd13143fabf784c80b901586452f9990b7e14,   0, COINBASE_OUTPUT
   5523.296671 T, ba4bf550d8dda2987ed175295128133056f6a54e76cec1af8ff567ebabb68d5e,   0, COINBASE_OUTPUT
   5523.303949 T, 027c9dd268ef967ff688c1ebd48cdabdbec2b76f62ecd6a2625726db6cb98259,   0, COINBASE_OUTPUT
   5523.306375 T, 087294b16055be95eae25f1572d15e28c561d8d44c2bcc375934ea2b63ced42f,   0, COINBASE_OUTPUT
```

**Comment by hansieodendaal** [comment]
> 
> 
> Very interesting - do you have the wallet logs please? Did this only happen with coinbase UTXOs or all?

Uploaded ^^
